### ISSUE TITLE
Question about abbreviations process

### ISSUE THREAD

**Issue by xichie** [LeaperOvO/KnowLog#4]
The paper replaces the abbreviations with their full names as shown in Figure 1. However, it seems that no processing was done in pre-training stage, and it exclusively arbitrarily masked the abbreviations. Is my understanding correct?
